src/dev.rs

```diff
@@ -352,12 +354,7 @@ impl<F: Field + Group> Assignment<F> for MockProver<F> {
         self.regions.push(self.current_region.take().unwrap());
     }
 
-    fn enable_selector<A, AR>(
-        &mut self,
-        annotation: A,
-        selector: &Selector,
-        row: usize,
-    ) -> Result<(), Error>
+    fn enable_selector<A, AR>(&mut self, _: A, selector: &Selector, row: usize) -> Result<(), Error>
```

Comment:
If we ignore the annotation here even in dev, we should probably just remove it.

Reply:
I suspect it'll come in handy later, but removing the parameter would be disruptive right now. I'd rather wait to remove until we're sure it's useless, because if we remove now then downstream callers have to remove the annotations only to potentially put them back if it turns out they're useful. Out of scope for this PR I think.